src/app/services/species.service.ts

```diff
@@ -22,22 +21,19 @@ export class SpeciesService {
 
   constructor(public http: HttpClient) { }
 
-  getAll(): Observable<ApiResponse>{
+  getAll(): Observable<ApiResponse> {
     return this.http.get<ApiResponse>(this.apiURL);
   }
-  
-  getById(id: number): Observable<any>{
+
+  getById(id: number): Observable<any> {
     return this.http.get(this.apiURL + id);
   }
 
-  getNextPage(){
-    
+  getNextPage() {
```

Comment:
If these aren't defined or returning anything, then remove them. This is part of making sure the code is test-driven - it shouldn't exist without a test.

Reply:
I will remove them